mlrun/runtimes/base.py

```diff
@@ -440,6 +440,7 @@ def run(
         else:
             # single run
             try:
+                self.verify_run_params(run.spec.parameters)
                 resp = self._run(run, execution)
                 if watch and self.kind not in ["", "handler", "local"]:
                     state = run.logs(True, self._get_db())
```

Comment:
I am not quite sure about the location of that.
lets add a test that actually lists the project runs and verify that the run wasn't recorded.